src/clients/base.rs

```diff
@@ -264,7 +264,7 @@ where
     ///
     /// Parameters:
     /// - artist_id - the artist ID, URI or URL
-    /// - album_type - 'album', 'single', 'appears_on', 'compilation'
+    /// - include_groups - 'album', 'single', 'appears_on', 'compilation'
```

Comment:
I think doc should be updated. For example

A list of album type that will be used to filter the response. If not supplied, all album types will be returned.

Reply:
I have updated the document with your example.